### DISCUSSION TITLE
How it works?

### DISCUSSION THREAD

**Discussion by serg883** [kibook/spooner#9] [Q&A] [answered]
Hi! Thanks for great work. Spooner is awesome. It is a very useful stuff for Redm! But I can't activate new version of sponner. In older versions it was button 'Delete'. It work correct. New version ist't work, I don't know why. May be it need some another resources? or comand? or permissions? sorry for bad english

**Comment by kibook** [comment, marked as answer]
It does require another resource, my [uiprompt](https://github.com/kibook/redm-uiprompt) library. If you haven't installed it, that might be why it's not working.

**Reply by serg883** [reply]
Now it works!! Thank you very much!!!
